Script/Packages/Atomic/Input.json

```diff
@@ -4,7 +4,8 @@
 	"classes" : ["Input"],
 	"overloads" : {
 		"Input" : {
-			"BindButton" : ["UIButton", "int"]
+			"BindButton" : ["UIButton", "int"],
+			"GetTouch" : ["unsigned"]
```

Comment:
This shouldn't actually be doing anything, because TouchState isn't a scriptable class, so will get ignored.  

So, we'll need to add the custom marshaller to the typescript/haxe defs, and extra points for defining an interface for the object properties/keys it returns